java/reflection/BaseType.java

```diff
@@ -0,0 +1,29 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Are you going to move these files to `java/com/google/flatbuffers/reflection` ?
Also, please add a line to `reflection/generate_code.sh` to automatically generate these files.